components/Section/Section.tsx

```diff
@@ -84,9 +84,9 @@ export interface SectionProps {
   children: React.ReactNode;
 }
 
-export const Section = ({ bg, children }: SectionProps) => {
+export const Section = ({ bg, children, ...props }: SectionProps) => {
```

Comment:
```suggestion
export const Section = ({ bg, children, ...props }: SectionProps & BoxProps) => {
```

But it first import `BoxProps` and the top of the file.